Configuration is invalid - found 2 configuration error(s), and 0 rule(s).
semgrep error: Detected duplicate key
  --> rules/invalid-rules/invalid-metavariable-regex.yaml:6
5  |     - metavariable-regex:
6  |         metavariable: $HOST
7  |         regex: '192.168\.\d{1,3}\.\d{1,3}'
8  |         metavariable: $MODULE
9  |         regex: (boto|boto3)
10 |   message: "Boto3 connection to internal network"
11 |   languages: [python]

Detected duplicate key name, one of ['metavariable', 'regex'].

'SemgrepCoreError' object has no attribute 'color'
Traceback (most recent call last):
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/commands/wrapper.py", line 46, in wrapper
    func(*args, **kwargs)
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/commands/scan.py", line 879, in scan
    output_handler.handle_semgrep_errors(config_errors)
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/output.py", line 218, in handle_semgrep_errors
    self._handle_semgrep_error(err)
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/output.py", line 258, in _handle_semgrep_error
    logger.error(error.format_for_terminal())
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/error.py", line 88, in format_for_terminal
    return format_for_terminal(str(self), self.color, self.level.name)
AttributeError: 'SemgrepCoreError' object has no attribute 'color'
